app/browser/dataset-browse.tsx

```diff
@@ -353,12 +353,12 @@ const NavItem = ({
         Boolean
       )
     );
-    const nextIndex = filters.findIndex((f) => f.iri === next.iri);
-    const newFilters = nextIndex === 0 ? [] : filters.slice(0, 1);
+    // const nextIndex = filters.findIndex((f) => f.iri === next.iri);
+    const newFilters = filters.filter((d) => d.iri !== next.iri);
 
-    return (
-      "/browse?" + newFilters.map(encodeFilter).join("&") + `&${extraURLParams}`
-    );
+    return `/browse/${newFilters
+      .map(encodeFilter)
+      .join("/")}?${extraURLParams}`;
```

Comment:
Why no more theme & organisation filter ?

Reply:
If I remember correctly, this was needed to avoid the below behavior and to make it possible to only de-select one filter and make the other one stay :)


https://github.com/visualize-admin/visualization-tool/assets/52032047/2d65079d-8f6d-43ad-8182-9ed5f4f4a5f3

Reply:
But the filters still work :D